samples/tab-channel-group-quickstart/js/src/index.js

```diff
@@ -5,10 +5,10 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './components/App';
-import { Provider, themes } from '@fluentui/react-northstar' //https://fluentsite.z22.web.core.windows.net/quick-start
+import { FluentProvider, teamsDarkTheme } from '@fluentui/react-components' //https://fluentsite.z22.web.core.windows.net/quick-start
```

Comment:
Let's remove or add relevant URL in place of below:
//https://fluentsite.z22.web.core.windows.net/quick-start